frontend/src/components/dashboard/aliases/AliasList.tsx

```diff
@@ -110,14 +117,19 @@ export const AliasList = (props: Props) => {
       <li
         className={styles["alias-card-wrapper"]}
         key={alias.address + isRandomAlias(alias)}
+        id={encodeURIComponent(alias.full_address)}
       >
         <Alias
           alias={alias}
           user={props.user}
           profile={props.profile}
           onUpdate={onUpdate}
           onDelete={() => props.onDelete(alias)}
-          isOpen={openAlias !== undefined && openAlias.id === alias.id}
+          isOpen={
+            openAlias !== undefined &&
+            openAlias.id === alias.id &&
+            openAlias.mask_type === alias.mask_type
```

Comment:
I realised that theoretically, random and custom masks can have the same IDs (since they're in different database tables). So to be sure, I also added a check on the mask type.